src/vs/editor/browser/viewParts/selections/selections.ts

```diff
@@ -402,21 +400,6 @@ export class SelectionsOverlay extends DynamicViewOverlay {
 	}
 }
 
-registerThemingParticipant((theme, collector) => {
-	const editorSelectionColor = theme.getColor(editorSelectionBackground);
-	if (editorSelectionColor) {
-		collector.addRule(`.monaco-editor .focused .selected-text { background-color: ${editorSelectionColor}; }`);
-	}
-	const editorInactiveSelectionColor = theme.getColor(editorInactiveSelection);
-	if (editorInactiveSelectionColor) {
-		collector.addRule(`.monaco-editor .selected-text { background-color: ${editorInactiveSelectionColor}; }`);
-	}
-	const editorSelectionForegroundColor = theme.getColor(editorSelectionForeground);
-	if (editorSelectionForegroundColor && !editorSelectionForegroundColor.isTransparent()) {
```

Comment:
This specific rule cannot be moved to CSS because its condition is `if (editorSelectionForegroundColor && !editorSelectionForegroundColor.isTransparent())`. This cannot be expressed in CSS.